pkg/scheduler/framework/plugins/nodevolumelimits/csi.go

```diff
@@ -78,6 +79,7 @@ func (pl *CSILimits) EventsToRegister() []framework.ClusterEventWithHint {
 	return []framework.ClusterEventWithHint{
 		{Event: framework.ClusterEvent{Resource: framework.CSINode, ActionType: framework.Add}},
 		{Event: framework.ClusterEvent{Resource: framework.Pod, ActionType: framework.Delete}},
+		{Event: framework.ClusterEvent{Resource: framework.PersistentVolumeClaim, ActionType: framework.Add}},
```

Comment:
I'll take on implementing it as part of https://github.com/kubernetes/kubernetes/issues/118893.